### PR TITLE
Generate correct serde bound for GenesisConfig

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: srml/support/procedural/src/storage/transformation.rs [hunk 1]
```diff
@@ -239,9 +239,15 @@ fn decl_store_extra_genesis(
 				has_trait_field = true;
 			}
 
-			serde_complete_bound.insert(type_infos.value_type);
+			let value_type = &type_infos.value_type;
+			serde_complete_bound.push(quote!( #value_type ));
 			if let DeclStorageTypeInfosKind::Map { key_type, .. } = type_infos.kind {
```
Comment (thiolliere) [inline srml/support/procedural/src/storage/transformation.rs]: also do doublemap. and maybe add more test:
if I remove Foo test and replace by `pub Foo get(foo) config(): double_map u32, blake2_256(T::BlockNumber) => Option<u32>;`
but if Foo is there is works, so maybe add more associated type

Comment (bkchr) [reply srml/support/procedural/src/storage/transformation.rs]: What?

Comment (bkchr) [reply srml/support/procedural/src/storage/transformation.rs]: Added a test for double map.

Comment (thiolliere) [reply srml/support/procedural/src/storage/transformation.rs]: Yes but the test you added already works without the add of the bound of the second key as the bound is already explicited by the value.

Comment (bkchr) [reply srml/support/procedural/src/storage/transformation.rs]: Modified again, the new double map test failed on master.

Comment (thiolliere) [reply srml/support/procedural/src/storage/transformation.rs]: oh yes way simpler :-)

**Pull request merged**